routers/routes/routes.go

```diff
@@ -466,6 +466,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Post("/title", repo.UpdateIssueTitle)
 				m.Post("/content", repo.UpdateIssueContent)
 				m.Post("/watch", repo.IssueWatch)
+				m.Post("/add_time", repo.AddTimeManual)
```

Comment:
add subgroup `/times` with `/add`, `/stop`, and `/cancel` as these all than can be added route condition to check if user is allowed to add time